.env.example

```diff
@@ -14,6 +14,7 @@ FEATURE_COMPARISON_TABLE=SE
 FEATURE_CAR_CANCELLATION=SE
 FEATURE_CAR_PRICE_MATCHING=SE
 FEATURE_AB_TEST_PURCHASE_FLOWS=SE
+FEATURE_INTERCOM=NO,DK
```

Comment:
Just checking - was the idea to remove only from Sweden?

Reply:
yes exactly